atlasdb-service-server/src/test/java/com/palantir/server/TransactionRemotingTest.java

```diff
@@ -255,12 +286,12 @@ public void testRaw2() throws JsonProcessingException {
 
     private void setupFooStatus1(String table) {
         TransactionToken txId = service.startTransaction();
-        service.put(txId, new TableCellVal(table, getSweepPriorityTableContents()));
+        service.put(txId, new TableCellVal(table, getSweepPriorityTableContents("foo")));
         service.commit(txId);
     }
 
-    private Map<Cell, byte[]> getSweepPriorityTableContents() {
```

Comment:
Very optional nit: it would have been quite alright to keep this method around, defaulting to `getSweepPriorityTableContents("foo")`.